join-chat-to-teams-meeting/Chat Experience with File Sharing/README.md

```diff
@@ -0,0 +1,45 @@
+---
+page_type: sample
+languages:
+- javascript
+products:
+- azure
+- azure-communication-common
+- azure-communication-chat
+- azure-communication-calling
+- azure-communication-identity
+---
+
+# Teams Interop Chat App with File Attachment Support
+
+This code sample is explained as part of [this tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/chat-interop/meeting-interop-features-file-attachment). Please make sure you have gone through it first.
+
+## Prerequisites
+- An active Communication Services resource. [Create a Communication Services resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource).
+- A [Teams deployment](https://docs.microsoft.com/deployoffice/teams-install)
+
+## Code Structure
+
+- **./index.html:** to configure a basic layout that will allow the user to join a teams meeting.
+- **./client.js:** contain the application logic.
+
+## Setup
+
+1. Clone this sample
+3. Run `npm install`
+4. Grab the connection string from your communication resource and replace it at line #36 in client.js ```const connectionString = "<SECRET_CONNECTION_STRING>";```.
```

Comment:
Let make this an actual link to the line in the `client.js` makes it easier to showcase

Reply:
sure, but it would hard code to main branch and if you click on it now it would be 404

Reply:
actually, relative path works great with line numbers, updated